src/packer.pkr.hcl

```
@@ -1,3 +1,16 @@
+variable "winrm_password" {
+  default     = false
+  description = "The password used to set the Administrator password and connect to the instance via WinRM."
```

The only thing Packer does is connect with this password. The password is set in the bootstrap user-data which is not managed by Packer.

```suggestion
  description = "The password used to connect to the instance via WinRM."
```

---

resolved with [af9d5cd](https://github.com/cisagov/windows-server-packer/pull/10/commits/af9d5cd82a54be1cbe0d85865ac1710c62fb80f1)